app/models/split.rb

```diff
@@ -112,7 +134,9 @@ def create_decision!(params = {})
   end
 
   def registry
-    if try(:feature_incomplete?) # This is a virtual attribute provided by the with_feature_incomplete_knockouts_for scope
+    if try(:remote_kill_override_to) # These are virtual attributes provided by the for_app_build scope
+      knock_out_weightings(super, to: remote_kill_override_to)
+    elsif try(:feature_incomplete?) # These are virtual attributes provided by the for_app_build scope
       knock_out_weightings(super)
```

Comment:
I could see a very different world in which you somehow managed to do a `COALESCE(override_to, 'false')` somewhere in a very different set of queries. But in order to get to that place, you'd need to wrap some kind of polymorphic blanket around `FeatureCompletion`s and `RemoteKill`s. And while they are conceptually similar (both providing ranges over which assignments should/should not apply under certain conditions -- the distinction of future assignments overriding vs not overriding being one of the bigger differences) you'd have to solve the exact kind of overlapping prioritization problem that we are avoiding with `RemoteKill`s. So, not saying we should try to go there, but at the same time at the back of my brain there's a little engineer who wants a simpler / more elegant solution where perhaps there is nothing truly viable.

Reply:
sometimes you just wanna express a prioritized set of conditional actions. :) I'm saying here explicitly that remote kill override_to values will win over feature incomplete `false` if they are to different variants, which is actually a nice thing to see on the page, I think.

Reply:
(because both of these things can be true at once, so it should be clear to a reader that the remote_kill is going to take precedence)